Upgrade to 3.14 (#28)
stolostron/gatekeeper-operator: docs/upgrading-gatekeeper.md

## 1. Set desired Gatekeeper version and commit

> Change ./REPLACES_VERSION and ./VERSION
> Example: 
> If you want to upgrade from 3.14.0 to 3.15.0
> ***REPLACES_VERSION*** file
> ```text 
> 3.14.0
> ```
> ***VERSION*** file
> ```text 
> 3.15.0
> ```
> 

Reviewer: Providing commands like `releasing.md` does would help here to keep users at the shell:
```suggestion
Update ./REPLACES_VERSION and ./VERSION files.

For example, if you want to upgrade from the current version to 3.15.0:
```shell
cat VERSION > REPLACES_VERSION
printf "3.15.0" > VERSION
```
```

```shell

> GATEKEEPER_PREV_VERSION=v(cat REPLACES_VERSION)

Reviewer 2: :cat2: 
```suggestion
GATEKEEPER_PREV_VERSION=v$(cat REPLACES_VERSION)
```

> GATEKEEPER_PREV_VERSION=v(cat REPLACES_VERSION)

Author: Thanks!!

> GATEKEEPER_PREV_VERSION=v$(cat REPLACES_VERSION)

Reviewer: This will need to strip the metadata from the operator version:
```suggestion
GATEKEEPER_PREV_VERSION=v$(cut -d '-' -f 1 REPLACES_VERSION)
```

GATEKEEPER_PREV_VERSION=$(awk '/^GATEKEEPER_VERSION \?= .*/ {print $3}' Makefile)
GATEKEEPER_VERSION=<DESIRED_VERSION>
sed -i "s/GATEKEEPER_VERSION ?= .*/GATEKEEPER_VERSION ?= ${GATEKEEPER_VERSION}/" Makefile

> sed -i "s/VERSION ?= .*/VERSION ?= ${GATEKEEPER_VERSION}/" Makefile

Reviewer 2: Is this doc still accurate after @dhaiducek's changes in #29 to use the version files?
